### DISCUSSION TITLE
Custom readTimeOut and writeTimeOut for Traefik ingress

### DISCUSSION THREAD

**Discussion by pranaysahith** [k3s-io/k3s#5926] [Q&A]
How can we increase the default readTimeout, writeTimeout and idleTimeout in Traefik ingress while using k3s?

**Comment by brandond** [comment]
https://rancher.com/docs/k3s/latest/en/helm/#customizing-packaged-components-with-helmchartconfig

The available values can be found here: https://github.com/traefik/traefik-helm-chart/blob/master/traefik/values.yaml

**Comment by pranaysahith** [comment]
There are no default values in the vaules.yaml file of the helm chart to override the default timeouts. We need to pass the cli arguments as "--entrypoints.<name>.transport.respondingtimeouts.readtimeout=600" using https://github.com/traefik/traefik-helm-chart/blob/master/traefik/values.yaml#L253.

**Reply by brandond** [reply]
Look more closely - it sounds like you want to use the `additionalArguments` value?
```
# Configure Traefik static configuration
# Additional arguments to be passed at Traefik's binary
# All available options available on https://docs.traefik.io/reference/static-configuration/cli/
## Use curly braces to pass values: `helm install --set="additionalArguments={--providers.kubernetesingress.ingressclass=traefik-internal,--log.level=DEBUG}"`
additionalArguments: []
```
